pkg/domain/entities/types/container_ps.go

```diff
@@ -118,3 +118,31 @@ func (l ListContainer) USERNS() string {
 func (l ListContainer) UTS() string {
 	return l.Namespaces.UTS
 }
+
+func (l ListContainer) Commands() []string {
+	return l.Command
```

Comment:
You need to list these in `podman-ps.1.md` under `--format`

Reply:
It is already there:

https://github.com/containers/podman/blob/b18dcccb28e3894b2f8ec914a13024b39953b3a8/docs/source/markdown/podman-ps.1.md?plain=1#L76C1-L76C70

Reply:
@mheon  I fixed all manpage issues.